index.bs

```diff
@@ -1550,6 +1571,26 @@ To <dfn>parse aggregation keys</dfn> given an [=ordered map=] |map|:
 
 Issue: Determine whether to limit [=string/length=] or [=string/code point length=] for |key| above.
 
+To <dfn>obtain effective windows</dfn> given an [=attribution source/source type=] |sourceType|,
+a [=moment=] |sourceTime|, and a [=duration=] |eventReportWindow|:
+
+1. Let |deadlines| be «» if |sourceType| is "<code>[=source type/event=]</code>", else « 2 days, 7 days ».
+1. [=list/Remove=] all elements in |deadlines| that are greater than |eventReportWindow|.
+1. [=list/Append=] |eventReportWindow| to |deadlines|.
+1. Let |lastEnd| be |sourceTime|.
+1. Let |windows| be «».
+1. [=list/iterate|For each=] |deadline| in |deadlines|:
+    1. Let |window| be a new [=event-level report window=] whose items are
+
+    : [=event-level report window/start=]
+    :: |lastEnd|
+    : [=event-level report window/end=]
+    :: |lastEnd| + |deadline|
+
+    1. [=list/Append=] |window| to |windows|.
+    1. Let |lastEnd| be |deadline|.
```

Comment:
I think the formatting here is causing numbering to restart at 1 in the rendered spec. Maybe the lines starting with `:` and `::` need to be indented more?

Reply:
Indenting fixed it.